### PR TITLE
Add oneOfSourceSuffixes option

### LINKED ISSUE #34: Одновременное использование file.post.css и file.css
причина https://github.com/awinogradov/enb-postcss/blob/master/techs/enb-postcss.js#L27

https://github.com/awinogradov/enb-postcss/blob/master/techs/enb-postcss.js#L31
фильтрация не происходит, т.к. basename равен .../file.post и .../file соответственно

### PR DESCRIPTION
**Pull request by tadatuta**
Closes #34 

Possible values of `oneOfSourceSuffixes` option:
```js
null or any other falsy value // as if there was no oneOfSourceSuffixes option
'css' // keep just files with `css` tech
['post.css', 'css'] // keep just one of these techs, order is significant
[['post.css', 'css'], 'ie.css', 'styl'] // keep (post.css || css) + ie.css + styl
[['post.css', 'css'], ['ie.post.css', 'ie.css']] // (post.css || css) + ('ie.post.css' || 'ie.css')
[['post.css', 'css'], 'blah', ['ie.post.css', 'ie.css']] // (post.css || css) + blah + ('ie.post.css' || 'ie.css')
```

### CODE REVIEW

**Comment by awinogradov** [comment]
Why this 
``` js
[['post.css', 'css'], ['ie.post.css', 'ie.css']] // (post.css || css) + ('ie.post.css', 'ie.css')
[['post.css', 'css'], 'blah', ['ie.post.css', 'ie.css']] // (post.css || css) + blah + ('ie.post.css', 'ie.css')
```

not this?
``` js
[['post.css', 'css'], ['ie.post.css', 'ie.css']] // (post.css || css) + ('ie.post.css' || 'ie.css')
[['post.css', 'css'], 'blah', ['ie.post.css', 'ie.css']] // (post.css || css) + blah + ('ie.post.css' || 'ie.css')
```

**Comment by tadatuta** [comment]
Sure, you are right, it's a typo. Fixed.

**Comment by awinogradov** [comment]
👍

File: techs/enb-postcss.js [hunk 1]
```diff
@@ -11,31 +11,16 @@ module.exports = buildFlow.create()
...
-
-                if (added[basename]) {
-                    return false;
-                }
-
-                added[basename] = true;
-
-                return true;
-            }).map(function (file) {
+            css = (this._oneOfSourceSuffixes ?
```
Comment (aliosv) [inline techs/enb-postcss.js:21]: лишняя проверка

**Pull request merged**